src/pages/patientView/PatientViewPage.tsx

```diff
@@ -243,10 +243,10 @@ export default class PatientViewPage extends React.Component<IPatientViewPagePro
                                 )
                             }
                         </span>
-                        {patientViewPageStore.hasMutationalSignatureData.isComplete &&
+                        {patientViewPageStore.hasMutationalSignatureData.result &&
```

Comment:
why change from isComplete to result?

Reply:
The info should only be shown when the result is true. Not when isComplete

Reply:
I see, so it's boolean.

Reply:
maybe add ===true just to make it super clear.  otherwise, it seems like we're using it as a proxy for isComplete

Reply:
Updated